drone_localization/streamlit/gallery_app.py

```
@@ -0,0 +1,384 @@
+                    if coordinates:
+                        # Формат coordinates должен быть объектом с tl_E, tl_N, br_E, br_N
+                        payload["coordinates"] = {
+                            "tl_E": coordinates["tl_E"],
+                            "tl_N": coordinates["tl_N"],
+                            "br_E": coordinates["br_E"],
+                            "br_N": coordinates["br_N"]
+                        }
+                    if metadata:
+                        payload["metadata"] = metadata
+
+                    response = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)